src/components/TreeControls/TreeControls.vue

```diff
@@ -178,8 +193,17 @@ export default {
         else this.myPopperClass = "hide-scaffold-colour-popup";
       },
     },
+    filterText: {
+      handler: function (value) {
+        if (this.$refs.regionTree) this.$refs.regionTree.filter(value);
+      },
+    },
   },
   methods: {
+    filterNode: function(value, data) {
+      if (!value) return true;
+      return data.label ? data.label.toLowerCase().includes(value) : false;
```

Comment:
I found this does not work for uppercase values. I've fixed this here: https://github.com/alan-wu/map-utilities/pull/2

Reply:
Thanks a lot!!!